general/development/tools/behat/browsers.md

```diff
@@ -0,0 +1,76 @@
+
+### Examples
+
+```console title="Selenium in Linux (firefox by default + chrome)"
+   java -jar /opt/selenium-server-standalone.jar -Dwebdriver.chrome.driver=/opt/chromedriver
+```
+
+```console title="Selenium in OSx (firefox & safari by default + chrome)"
+  java -jar /Users/moodle/Downloads/selenium-server-standalone.jar -Dwebdriver.chrome.driver=/Users/moodle/Downloads/chromedriver
+```
+
+```console title="Selenium in Windows (started using git bash) (firefox by default + chrome + internet explorer)"
+  java -jar /c/seleniumdrivers/selenium-server-standalone.jar -Dwebdriver.chrome.driver=/c/seleniumdrivers/chromedriver.exe -Dwebdriver.ie.driver=/c/seleniumdrivers/IEDriverServer.exe
+```
+
+```console title="PhantomJS"
+  /path/to/your/phantomjs/bin/phantomjs --webdriver=4444
+```
+
+## Compatibility
+
+Not all the drivers can execute all of Moodle's step definitions; we tagged the step definitions that are using features not supported by all browsers and also limitations that some browsers have; refer to the following table to know which browsers can run which tags:
+
+<!-- cspell:ignore zerosize -->
+| | File uploads (@_file_upload) | Browser dialogs (@_alert)                                         | Switch window (@_switch_window) | Switch frame (@_switch_iframe) | Bugs in chrome (@skip_chrome_zerosize) | Bug in phantomjs (@_bug_phantomjs) |
+|------------------------------|-------------------------------------------------------------------|---------------------------------|--------------------------------|----------------------------------------|------------------------------------|
+| Firefox                      | Yes | Yes | Yes | Yes | Yes | Yes |
+| Chrome              | Yes | Yes | Yes | Yes | NO (see [MDL-71108](https://tracker.moodle.org/browse/MDL-71108)) | Yes |
+| Internet Explorer          | Yes | Yes | No | Yes | Yes | Yes |
+| Safari          | Yes | No | No | Yes | Yes | Yes |
+| PhantomJS          | No | No | Yes | Yes | Yes | No |
+
+Note that, to skip some tag, you must prepend it with the <tt>~</tt> (logical NOT) character. Examples:
+
+- Run all tests but `@_alert</tt> ones: <tt>--tags '~@_alert'`
+- Run all chrome tests but `@skip_chrome_zerosize` ones: `--tags '@JavaScript&&~@skip_chrome_zerosize'`
+
+## Working combinations of OS+Browser+selenium
+
+As OS, Browsers and Selenium keeps updating, some combination of OS+Browser+Selenium will not work on specific moodle version.
+
+We try to support the latest version of these combinations but they are not always BC and hence may not work with older releases. Please refer to [Acceptance testing/Browsers/Working combinations of OS+Browser+selenium](https://docs.moodle.org/dev/Acceptance_testing/Browsers/Working_combinations_of_OS+Browser+selenium) to ensure you have correct combination of them to run acceptance test.
```

Comment:
Will this page (https://docs.moodle.org/dev/Acceptance_testing/Browsers/Working_combinations_of_OS+Browser+selenium) be migrated too? If that's the case, I would suggest creating an issue for it (just to guarantee we don't forget about it).